### ISSUE TITLE
Extraneous EXPECTED_REWIRINGs in successive compilations

### ISSUE THREAD

**Issue by notmgsk** [rigetti/quilc#129]
Not exactly a bug

```common-lisp
CL-USER> 
(quil::print-parsed-program
 (quil::compiler-hook 
  (quil::compiler-hook 
   (quil::parse-quil-string
"
RX(pi) 1
HALT") 
   (quil::build-nq-linear-chip 3))
  (quil::build-nq-linear-chip 3)))
PRAGMA EXPECTED_REWIRING "#(0 1 2)"
PRAGMA EXPECTED_REWIRING "#(0 1 2)"
RX(pi) 1
PRAGMA CURRENT_REWIRING "#(0 1 2)"
HALT
NIL
```

**Comment by ecp-rigetti** [comment]
I'd start chasing this either in `src/compilation-methods.lisp` (and `compiler-hook` specifically) or in `src/cfg.lisp`. My memory is that these lines get inserted only after block compilation is actually done—so either directly after resolving the compilation in a block, or when the CFG is serialized back into a `parsed-program`.

**Comment by ecpeterson** [comment]
Closed as of #159 .
